packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -336,3 +336,12 @@ webSocketApi.addRoute('sendmessage', {
   }),
 });
 ```
+
+In addition, you can enable API key authentication to your WebSocket api:
```

Comment:
Can you add a separate section called 'api keys' and explain them a little more.